admin1.php

```diff
@@ -33,29 +33,27 @@
 			
 			<!-- login box -->
 			<div class = "login_box">
-			
-				<b><p style = "color: black; font-size: 2vw; margin: 5.5vw 0vw 0vw 3.7vw;">Add User</p></b>
-
+				
+				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Add User</p></b>
+				
 				<!-- Form for input fields -->
 				<h3><form method = "post" id= "01_login">
-
-					<p style = "margin: -2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Username:</p>
-					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "username" placeholder="Your Username..."/>
-
-					<p style = "margin: 1.5vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Password:</p>
-					<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "password" name = "password" placeholder="Your Password..." id="myInput"/>
 						
-					<p style = "margin: 1.3vw 0vw -1.3vw 5.3vw; font-size: 1vw;">Show Password</p>
-					
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 4vw 0vw 0vw 4.2vw;" type = "text" name = "username" placeholder="Username..."/>

```

Comment:
below is where I have adjusted the input fields' margins and other style elements.